unity/Holo/Assets/GFX/Shaders/HoloDataFlow.shader

```diff
@@ -197,7 +202,9 @@
 				fixed4 col = i.col;
 				col.a = i.col.a;
 
-				if(_DisplayMode == 1)
+				if(_DisplayMode == 0)
```

Comment:
This should be solved using `#ifdef XXX` and "shader keywords", like I did with CLIPPING_PLANE. This way, the comparisons `if(_DisplayMode == xxx)` will not be done at runtime, at every shader execution.